From bb20c565417a2c2ab274b254e6238657c5d8c73a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C5=BDidek?= <mzidek@redhat.com>
Date: Thu, 26 Oct 2017 17:12:17 +0200
Subject: [PATCH 11/79] MAN: GPO Security Filtering limitation
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Note in the man pages that current version of SSSD does not support
host entries in the 'Security filtering' list.

Resolves:
https://pagure.io/SSSD/sssd/issue/3444

Reviewed-by: Fabiano Fidêncio <fidencio@redhat.com>
---
 src/man/sssd-ad.5.xml | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/src/man/sssd-ad.5.xml b/src/man/sssd-ad.5.xml
index 08c1dd09fb829c6cffb416250b9b518668ec5790..649042d587de3d3600fff59866681e302c721af8 100644
--- a/src/man/sssd-ad.5.xml
+++ b/src/man/sssd-ad.5.xml
@@ -345,6 +345,13 @@ DOM:dom1:(memberOf:1.2.840.113556.1.4.1941:=cn=nestedgroup,ou=groups,dc=example,
                             particular user is allowed to logon to a particular
                             host.
                         </para>
+                        <para>
+                            NOTE: The current version of SSSD does not support
+                            host (computer) entries in the GPO 'Security
+                            Filtering' list. Only user and group entries are
+                            supported. Host entries in the list have no
+                            effect.
+                        </para>
                         <para>
                             NOTE: If the operation mode is set to enforcing, it
                             is possible that users that were previously allowed
-- 
2.15.1

